tools/database/src/make_snapshot.rs

```diff
@@ -17,10 +21,22 @@ impl MakeSnapshotCommand {
     ) -> anyhow::Result<()> {
         let opener = NodeStorage::opener(home_dir, archive, store_config, None);
         let node_storage = opener.open_in_mode(Mode::ReadWriteExisting)?;
+        let columns_to_keep = if self.flat_state_only {
+            Some(&[
+                DBCol::DbVersion,
+                DBCol::BlockMisc,
+                DBCol::FlatState,
+                DBCol::FlatStateChanges,
+                DBCol::FlatStateDeltaMetadata,
+                DBCol::FlatStorageStatus,
```

Comment:
It would be nice to reuse the same list that is used in the implementation to make sure we don't diverge over time.